authui/src/authflowv2/botprotection/cloudflareTurnstile.ts

```diff
@@ -38,6 +38,10 @@ export class CloudflareTurnstileController extends Controller {
           }
         },
       });
+      for (let i = 0; i < this.widgetTarget.children.length; i++) {
+        const widget = this.widgetTarget.children[i];
+        widget.classList.add("flex");
+      }
```

Comment:
Why do we need this? Can we require the div to have `flex` already?

Reply:
> Why do we need this?

I just realized we don't need this. Lemme remove it.
Should be some UI debugging code 🙏 

> Can we require the div to have flex already?

No, because the div is dynamically rendered by cloudflare